src/sidebar/annotation-metadata.js

```diff
@@ -101,6 +101,24 @@ function isNew(annotation) {
   return !annotation.id;
 }
 
+/** Return `true` if the given annotation is public, `false` otherwise. */
+function isPublic(annotation) {
+  var isPublic = true;
+
+  if (!annotation.permissions) {
+    return null;
```

Comment:
If the status of the annotation is unknown, I think it would make sense to return `false`.